### DISCUSSION TITLE
Create news link (page repost)

### DISCUSSION THREAD

**Discussion by wonderplayer** [pnp/pnpcore#712] [Q&A] [answered]
Hello,

Was wondering, if it is possible in some way to create news link with PnP Core?
I see that there is REST API for that (example [here](https://sharepoint.handsontek.net/2020/11/01/post-sharepoint-news-link-programmatically/)), but didn't find any mention about it in PnP Core.

Thanks!

**Comment by jansenbe** [comment, marked as answer]
@wonderplayer : this is possible, there's not specific documentation on this but I think the test case will provide the needed information: https://github.com/pnp/pnpcore/blob/dev/src/sdk/PnP.Core.Test/SharePoint/PagesTests.cs#L2826-L2920

**Reply by wonderplayer** [reply]
Thanks a lot!
